R/drive_ls.R

```diff
@@ -25,7 +30,7 @@ drive_ls <- function(path = "~/", pattern = NULL, type = NULL, ...) {
 
   x <- list(...)
 
-  if (is.character(path)) {
+  if (is_path(path)) {
```

Comment:
A new helper! In many places we were equating "is character" to "is a character path" but we really need to also make sure it's NOT character marked as file id.